test/llvm.expect.ll

```diff
@@ -0,0 +1,195 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv %t.bc --spirv-ext=+SPV_INTEL_optimization_hints -spirv-text -o %t.spt
+; RUN: FileCheck < %t.spt %s --check-prefix=CHECK-SPIRV
+; RUN: llvm-spirv %t.bc --spirv-ext=+SPV_INTEL_optimization_hints -o %t.spv
+; RUN: llvm-spirv -r %t.spv -o %t.bc
+; RUN: llvm-dis < %t.bc | FileCheck %s --check-prefix=CHECK-LLVM
+
+; RUN: llvm-spirv %t.bc -spirv-text -o %t.spt
+; RUN: FileCheck < %t.spt %s --check-prefix=CHECK-SPIRV-NO-EXT
+; RUN: llvm-spirv %t.bc -o %t.spv
+; RUN: llvm-spirv -r %t.spv -o %t.bc
+; RUN: llvm-dis < %t.bc | FileCheck %s --check-prefix=CHECK-LLVM-NO-EXT
+
+; CHECK-SPIRV: 2 Capability OptimizationHintsINTEL
+; CHECK-SPIRV: 9 Extension "SPV_INTEL_optimization_hints"
+; CHECK-SPIRV: 5 Function
+; CHECK-SPIRV: 5 ExpectINTEL {{[0-9]+}} [[RES1:[0-9]+]] {{[0-9]+}} {{[0-9]+}}
```

Comment:
I think you should also check the rest of `ExpectINTEL` arguments, i.e. you should at least check expected value passed to it